src/routes/login/+page.svelte

```diff
@@ -1,22 +1,115 @@
 <script>
+    // Framework imports
+    import { getToastStore } from "@skeletonlabs/skeleton";
+
+    // Import 🐦
+    import { ConstantBackoff, handleAll, retry } from "cockatiel";
+
+    // Import backend urls
+    import { urls } from "$lib/util.js";
+
     // Definitions
-    let somethingWrong = false;
+    // Constants
+    const toastStore = getToastStore();
+
+    const mailRegexPattern =
+        /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+    // Enums
+    const btnIcon = {
+        locked: "🔐",
+        unlocked: "🔓",
+    };
 
     const indicatorStatus = {
-        none: "",
+        none: "border-surface-700",
         sucess: "input-success",
```

Comment:
Siehe Anmerkungen bei Email Input. https://tailwindcss.com/docs/configuration#important-modifier.
Success wird ja jetzt definitiv nicht benutzt. Würde ich jetzt rausnehmen.

```suggestion
        none: "",
```